gallery/visualtests/nxTabs.js

```diff
@@ -11,7 +11,52 @@ describe('NxTabs', function() {
     await browser.url('#/pages/NxTabs');
   });
 
-  const selector = '.gallery-example .nx-tabs';
+  const tabTileExampleSelector = '#nx-tab-tile-example .gallery-example-live',
+      tabTileNoHeaderExampleSelector = '#nx-tab-tile-no-header-example .gallery-example-live',
+      tabModalExampleSelector = '#nx-tab-modal-example',
+      tabModalNoHeaderExampleSelector = '#nx-tab-modal-no-header-example';
 
-  it('looks right', simpleTest(selector));
+  it('looks right in a Tile', simpleTest(tabTileExampleSelector));
+  it('looks right in a Tile with no header', simpleTest(tabTileNoHeaderExampleSelector));
+
+  describe('Check tab styles', function() {
+    const selector = '#test-tab-list .nx-tab-test:nth-child(2)';
+
+    it('has regular text and no border by default', simpleTest(selector));
+    it('has semi-bold text when hovered', hoverTest(selector));
+    it('has a light blue border and semi-bold text when clicked', clickTest(selector));
+    it('has a light blue border when focused', focusTest(selector));
+    it('has a light blue border and semi-bold text when focused and hovered', focusAndHoverTest(selector));
+  });
+
+  function simpleModalTest(exampleSelector) {
+    return async function() {
+      const openModalBtnSelector = `${exampleSelector} button`,
+          closeModalBtnSelector =
+              `${exampleSelector} .nx-footer .nx-btn-bar .nx-btn:not(.nx-btn--primary):not(.nx-btn-tertiary)`;
+
+      const openModalBtn = await browser.$(openModalBtnSelector);
+
+      await openModalBtn.scrollIntoView({ block: 'center' });
+      await openModalBtn.click();
+
+      const closeModalBtn = await browser.$(closeModalBtnSelector);
+
+      try {
+        // take image of entire viewport in order to capture the backdrop color
+        await browser.eyesSnapshot(null);
```

Comment:
For this test you probably don't really need the entire viewport

Reply:
> For this test you probably don't really need the entire viewport

https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/b9ae5a5e965c2e97e7cfc67c07f1bb9550864ba7

Reply:
> b9ae5a5

Pretty sure that needs to be `browser.eyesRegionSnapshot`

Reply:
> > [b9ae5a5](https://github.com/sonatype/sonatype-react-shared-components/commit/b9ae5a5e965c2e97e7cfc67c07f1bb9550864ba7)
> 
> Pretty sure that needs to be `browser.eyesRegionSnapshot`

Finger crossed! https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/cc742e19e8a9ba3e202868da89a06f3d763d39fa